ccloud-observability/docs/general-scenarios/request-rate-limits.rst

```diff
@@ -0,0 +1,32 @@
+.. _ccloud-observability-general-request-rate-limits:
+
+Request rate limits
+********************
+
+If request rate limits are hit, requests may be refused and clients may be throttled to keep the cluster stable. This throttling
```

Comment:
For the intro section:

1. Would it be helpful to first briefly define what a `request` is?  It may not be clear to new users.
2. What's `throttle` in this case?  Is it more than just refusing requests?
3. Please add a link to documented limits

Reply:
Addressed in https://github.com/confluentinc/examples/pull/952/commits/67d43da96fa0a7d3457747222007490f039bc799